lib/spack/external/py/_path/local.py

```diff
@@ -205,14 +205,14 @@ def remove(self, rec=1, ignore_errors=False):
             if rec:
                 # force remove of readonly files on windows
                 if iswin32:
-                    self.chmod(448, rec=1) # octcal 0700
+                    self.chmod(0o700, rec=1)
                 py.error.checked_call(py.std.shutil.rmtree, self.strpath,
                     ignore_errors=ignore_errors)
             else:
                 py.error.checked_call(os.rmdir, self.strpath)
         else:
             if iswin32:
-                self.chmod(448) # octcal 0700
+                self.chmod(0o700)
```

Comment:
Was there a problem with this? This file is part of an external package, so every time the external package gets updated, someone will overwrite all of these files with the latest version. You may want to contribute it upstream if you need it.